app/graphql/types/wallets/recurring_transaction_rules/object.rb

```diff
@@ -10,11 +10,15 @@ class Object < Types::BaseObject
 
         field :granted_credits, String, null: false
         field :interval, Types::Wallets::RecurringTransactionRules::IntervalEnum, null: true
+        field :method, Types::Wallets::RecurringTransactionRules::MethodEnum, null: false
         field :paid_credits, String, null: false
+        field :target_ongoing_balance, String, null: true
         field :threshold_credits, String, null: true
         field :trigger, Types::Wallets::RecurringTransactionRules::TriggerEnum, null: false
 
         field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+
+        delegate :method, to: :object
```

Comment:
It's not done automatically?

Reply:
nope, because `method` is a reserved keyword 😢